libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -501,6 +596,7 @@ bool AP_MotorsUGV::pre_arm_check(bool report) const
 // sanity check parameters
 void AP_MotorsUGV::sanity_check_parameters()
 {
+    _esc_backward_unlock = constrain_int16(_esc_backward_unlock, 0, 1);
```

Comment:
I don't think we actually need this range check, the code is only checking for == 0, there is no requirement that has to be zero or 1.